sources/engine/Stride.Rendering/Rendering/Images/ColorTransforms/ToneMap/ToneMapACESOperatorShader.sdsl

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Stride contributors (https://stride3d.net) and Silicon Studio Corp. (https://www.siliconstudio.co.jp)
+// Distributed under the MIT license. See the LICENSE.md file in the project root for more information.
+
+namespace Stride.Rendering.Images
+{
+    /// <summary>
+    /// The ACES tonemap operator.
+    /// </summary>
+    internal shader ToneMapACESOperatorShader : ToneMapCommonOperatorShader
+    {
+	    // ACES filmic tonemapper with highlight desaturation ("crosstalk").
+		// Based on the curve fit by Krzysztof Narkowicz.
+		// https://knarkowicz.wordpress.com/2016/01/06/aces-filmic-tone-mapping-curve/
+
+        override float4 Compute(float4 color)
+        {
+			float pixelLuminance = LuminanceUtils.Luma(color);
```

Comment:
Mix of tabs and space

Reply:
Resolving this, because it's the same as in the main branch.